build.gradle.kts

```diff
@@ -45,9 +46,8 @@ tasks.test {
   }
 }
 
-tasks.dokka {
-  outputFormat = "html"
-  outputDirectory = "$buildDir/javadoc"
+tasks.dokkaHtml.configure {
```

Comment:
I tried to merge what you had and what [the dokka 1.4 docs say](https://kotlin.github.io/dokka/1.4.0/user_guide/gradle/usage/)